Gruntfile.js

```diff
@@ -40,6 +40,27 @@ module.exports = function (grunt) {
         }
       }
     },
+    run: {
+      options: {
+      },
+      submodulesUpdate: {
+        cmd: 'git',
+        args: [
+          'submodule',
+          'update',
+          '--init'
```

Comment:
I believe we need to use the --remote switch to ensure the latest commit from the branch is used.